doc/devel/development_workflow.rst

```diff
@@ -167,6 +161,17 @@ You can achieve this by using
     git commit -a --amend --no-edit
     git push [your-remote-repo] [your-branch] --force-with-lease
 
+.. hint::
```

Comment:
```suggestion
.. tip::
```
IMHO slightly better admontion type (a hint is to make you aware of something, a tip has a touch of  recommendation added).